eslint.config.js

```diff
@@ -0,0 +1,113 @@
+"use strict";
+
+const js = require("@eslint/js");
+const { FlatCompat } = require("@eslint/eslintrc");
+const eslintConfigESLintCJS = require("eslint-config-eslint/cjs");
+const globals = require("globals");
+const reactPlugin = require("eslint-plugin-react");
+const reactRecommended = require("eslint-plugin-react/configs/recommended");
+const jsxA11yPlugin = require("eslint-plugin-jsx-a11y");
+const reactHooksPlugin = require("eslint-plugin-react-hooks");
+
+const compat = new FlatCompat({
+    baseDirectory: __dirname
+});
+
+const [jsxA11yRecommended] = compat.extends("plugin:jsx-a11y/recommended");
+
+module.exports = [
+    {
+        ignores: [
+            "node_modules/**",
+            "_site/**",
+            "src/assets/js/**",
+            "src/_data/**",
+            "src/_11ty/**",
+            ".stylelintrc.js"
+        ]
+    },
+    ...eslintConfigESLintCJS,
+    {
+        files: ["**/*.js"],
+        languageOptions: {
+            globals: {
+                ...globals.node
+            }
+        },
+        rules: {
+            ...js.configs.recommended.rules,
+
+            // Disable eslint-plugin-node rules from eslint-config-eslint
+            "no-process-exit": "off",
+            "func-style": "off",
+            "n/no-deprecated-api": "off",
```

Comment:
This will disable `eslint-plugin-n` rules for all files? Should we maybe keep these rules for files that run in Node.js (`eslint.config.js`, `.eleventy.js`, `tools/*.js`...)?